test/cppsuite/configs/hs_cleanup_stress.txt

```diff
@@ -54,30 +45,30 @@ workload_manager=
     ),
     populate_config=
     (
-        collection_count=10,
+        collection_count=100,
         key_count_per_collection=1000,
-        key_size=5,
-        thread_count=10,
-        value_size=100000
+        key_size=50,
+        thread_count=100,
+        value_size=1000000
     ),
     read_config=
     (
         op_rate=5ms,
-        ops_per_transaction=(max=100,min=1),
-        thread_count=8
+        ops_per_transaction=(max=1000,min=1),
+        thread_count=20
     ),
     update_config=
     (
-        op_rate=10ms,
+        op_rate=1ms,
         # Be careful to not aim too high with this config, if we fill the dirty cache and
         # all threads are trying to update, they'll get pulled into eviction and will get stuck.
-        ops_per_transaction=(max=20,min=0),
-        thread_count=10,
-        value_size=100000
+        ops_per_transaction=(max=10,min=0),
+        thread_count=100,
+        value_size=1000000
     )
 ),
 operation_tracker=
 (
-    enabled=true,
+    enabled=false,
     op_rate=20s
```

Comment:
We can remove this if the tracker is disabled

Reply:
bde1106ec